stdlib/public/core/Substring.swift

```diff
@@ -389,6 +389,13 @@ extension Substring.UTF8View: BidirectionalCollection {
     return _slice.distance(from: start, to: end)
   }
 
+  @inlinable
```

Comment:
this will need either availability or emitting into the client

Reply:
(that's why the test failed)

Reply:
Which should it have? It seems like it should be safe to emit into the client, but you've got more practice thinking about ABI compatibility than I do.

Reply:
`@_alwaysEmitIntoClient`